setup.py

```diff
@@ -98,10 +100,9 @@
             "gpu_exact_dups=nemo_curator.scripts.find_exact_duplicates:console_script",
             "deidentify=nemo_curator.scripts.find_pii_and_deidentify:console_script",
             "generate_statistics=nemo_curator.distributed_data_classification.generate_statistics:console_script",
```

Comment:
This needs to be removed since `generate_statistics` is gone now, right?

Reply:
Resolved via https://github.com/NVIDIA/NeMo-Curator/pull/58/commits/c6b5d9ed03dc790ab9b675b19de015478c686746